omero-ssh-systemd/Dockerfile

```diff
@@ -14,10 +14,16 @@ RUN sed -i \
 	's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' \
 	/etc/pam.d/sshd
 
+# To avoid error: sudo: sorry, you must have a tty to run sudo
+RUN sed -i -e "s/Defaults    requiretty.*/ #Defaults    requiretty/g" /etc/sudoers
+
 RUN useradd omero && \
 	echo 'omero:omero' | chpasswd && \
 	echo "omero ALL= (ALL) NOPASSWD: ALL" >> /etc/sudoers.d/omero
 
+RUN rm -f /lib/systemd/system/systemd*udev* ; \
```

Comment:
Can you add a link to https://bugzilla.redhat.com/show_bug.cgi?id=1046469#c11 so we can remove this workaround in future